config/crds/apis.kcp.dev_apibindings.yaml

```diff
@@ -312,6 +319,10 @@ spec:
                   - resource
                   type: object
                 type: array
+                x-kubernetes-list-map-keys:
+                - group
+                - resource
```

Comment:
This is wrong. It cannot be a map. group-resource won't be unique.

Reply:
with the upcoming permission claim changes this indeed will not be unique any more. there will be multiple group/resource entries, one per resourceName/Selector and separate verb list.